output-gazetteer.cpp

```diff
@@ -692,22 +692,20 @@ int output_gazetteer_t::process_way(osmid_t id, osmid_t *ndv, int ndc,
 
     /* Are we interested in this item? */
     if (places.has_data()) {
-        struct osmNode *nodev;
         int nodec;
 
+        assert(ndc > 0 && "no nodes to proccess");
```

Comment:
Really? Is this guaranteed anywhere further up in the processing chain? I would not assume that OSM data does not contain ways without nodes.